### PR TITLE
BZ-1276354: Tasks with ID higher than 1000 does not being completed. …

### PR DESCRIPTION
**Pull request by nmirasch**
…(change locale issue BZ:1261052)

### CODE REVIEW

File: uberfire-widgets/uberfire-widgets-commons/src/test/java/org/uberfire/ext/widgets/common/client/tables/ColumnPickerTest.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        verify( dataGrid ).addColumn( column, meta.getHeader() );
+        assertEquals( 1, dataGrid.getColumnCount() );
+        columnPicker.removeColumn( meta );
+        assertFalse( columnPicker.getColumnMetaList().contains( meta ) );
+        verify( dataGrid ).removeColumn( 0 );
+        assertEquals( 0, dataGrid.getColumnCount() );
+    }
+
+    @Test
+    public void testSortColumn() {
```
Comment (manstis) [inline uberfire-widgets/uberfire-widgets-commons/src/test/java/org/uberfire/ext/widgets/common/client/tables/ColumnPickerTest.java]: Does this test sorting? 

Shouldn't you check ```dataGrid.getColumn(0)``` is indeed ```column0``` and ```dataGrid.getColumn(1)``` is ```column1```?

**Comment by nmirasch** [comment]
I changed the test name, it's true, the test is over addColumns function and not sorting.

**Comment by manstis** [comment]
+1 to merge.

**Comment by csadilek** [comment]
0.5.x: a4f3bac9d05686bba66d32ae56fb5cfc09128da2

**Pull request closed without merging**